docs-src/python/how-to-set-a-cron-schedule-in-python.md

```diff
@@ -2,19 +2,20 @@
 id: how-to-set-a-cron-schedule-in-python
 title: How to set a Cron Schedule in Python
 sidebar_label: Cron Schedule
-description: Set a Cron Scheduler in Python
-tags:
-  - python
-  - how-to
+description: You can set each Workflow to repeat on a schedule with the cron_schedule option from either the start_workflow() or execute_workflow() asynchronous methods.
 ---
 
-You can set each Workflow to repeat on a schedule with the `cron_schedule` option from either the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) or [`execute_workflow()`](https://python.temporal.io/temporalio.client.Client.html#execute_workflow) asynchronous methods:
+You can set each Workflow to repeat on a schedule with the `cron_schedule` option from either the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) or [`execute_workflow()`](https://python.temporal.io/temporalio.client.Client.html#execute_workflow) asynchronous methods.
+
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-python/blob/main/your_cron_job/your_cron_dacx.py">View source code</a>
 
 ```python
-await client.start_workflow(
-    "your_workflow_name",
-    id="your-workflow-id",
-    task_queue="your-task-queue",
-    cron_schedule="* * * * *",
-)
+# ...
+    result = await client.execute_workflow(
+        LoopingWorkflow.run,
+        0,
```

Comment:
You know it's an argument to the workflow because you have workflow code and can see it takes an integer argument. But for someone reading these documents, it looks like an unexplained `0`. Maybe a cron workflow that doesn't accept an arg? Otherwise, let the user know it accepts an arg (either show workflow or add a comment or something).